# T6SS_RNASeq_manuscript
Study name: Transcriptome profiling of type VI secretion system core gene tssM mutant of Xanthomonas perforans highlights regulators controlling diverse functions ranging from virulence to metabolism

Authors: Sivakumar Ramamoorthy1, Michelle Pena1, Palash Ghosh1, Ying-Yu Liao2, Mathews Paret2, Jeffrey B Jones2 and Neha Potnis1*
1 Department of Entomology and Plant Pathology, Auburn University, AL 36849
2 Department of Plant Pathology, University of Florida, Gainesville, FL 32611

*Corresponding author: email: nzp0024@auburn.edu

Running title: T6SS regulation in Xanthomonas

The workflow (RNASeq_prokaryote_code.txt) was used to analyze RNASeq data collected on samples from in vitro experiment to identify differentially expressed genes among wild type X. perforans AL65 and tssM-i3* mutant at 8 and 16h post-inoculation in XVM2 medium. 

We used two parallel approaches epinatel (https://github.com/epinatel/Bacterial_RNAseq) and brnaseq (Barrick Lab) to analyze RNA-Seq data. The following code has been adopted and modified according to the computational resources available to us. The results from both analyses were compared. 
In the dataset presented in the manuscript, we have used the analysis obtained from epinatel pipeline.
The detailed output obtained from the workflow is included in TableS1 with the list of differentially expressed genes. 
